PluginGenerator/RulesPluginBuilder.cs

```diff
@@ -73,6 +74,13 @@ public RulesPluginBuilder SetSqaleFilePath(string filePath)
             return this;
         }
 
+        public RulesPluginBuilder SetRepositoryKey(string key)
+        {
+            RepositoryKeyUtilities.ThrowIfInvalid(key);
+            this.repositoryKey = key;
+            return this;
+        }
+
```

Comment:
Previously there wasn't a way to explicitly set the repository key; it was being calculated internally by the rule plugin builder. It now needs to be set explicitly by the caller.